images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -52,145 +59,144 @@ function Get-RustClippyVersion {
 }
 
 function Get-BindgenVersion {
-    return bindgen --version
+    return ((bindgen --version) -replace "^bindgen").Trim()
 }
 
 function Get-CbindgenVersion {
-    return cbindgen --version
+    return ((cbindgen --version) -replace "^cbindgen").Trim()
 }
 
 function Get-CargoAuditVersion {
-    return cargo-audit --version
+    return ((cargo-audit --version) -replace "^cargo-audit").Trim()
 }
 
 function Get-CargoOutdatedVersion {
-    return cargo outdated --version
+    return ((cargo outdated --version) -replace "^cargo-outdated").Trim()
 }
 
 function Get-PythonVersion {
-    return & python --version
+    return ((python --version) -replace "^Python").Trim()
 }
 
 function Get-PowershellCoreVersion {
-    return & pwsh --version
+    return ((pwsh --version) -replace "^PowerShell").Trim()
 }
 
 function Get-RubyVersion {
     $rubyVersion = $(ruby --version).split(" ")[1]
-    return "Ruby $rubyVersion"
+    return $rubyVersion
 }
 
 function Get-GoVersion {
     $(go version) -match "go(?<version>\d+\.\d+\.\d+)" | Out-Null
     $goVersion = $Matches.Version
-    return "Go $goVersion"
+    return $goVersion
 }
 
 function Get-KotlinVersion {
     $kotlinVersion = $((cmd /c "kotlinc  -version 2>&1") | Out-String).split(" ")[2]
-    return "Kotlin $kotlinVersion"
+    return $kotlinVersion
 }
 
 function Get-PHPVersion {
     ($(php --version) | Out-String) -match "PHP (?<version>\d+\.\d+\.\d+)" | Out-Null
     $phpVersion = $Matches.Version
-    return "PHP $phpVersion"
+    return $phpVersion
 }
 
 function Get-JuliaVersion {
     $juliaVersion = [regex]::matches($(julia --version), "\d+\.\d+\.\d+").Value
-    return "Julia $juliaVersion"
+    return $juliaVersion
 }
 
 function Get-LLVMVersion {
     $llvmVersion = [regex]::matches($(clang --version), "\d+\.\d+\.\d+").Value
-    return "LLVM $llvmVersion"
+    return $llvmVersion
 }
 
 function Get-PerlVersion {
     ($(perl --version) | Out-String) -match "\(v(?<version>\d+\.\d+\.\d+)\)" | Out-Null
     $perlVersion = $Matches.Version
-    return "Perl $perlVersion"
+    return $perlVersion
 }
 
 function Get-NodeVersion {
     $nodeVersion = $(node --version).split("v")[1]
-    return "Node $nodeVersion"
+    return $nodeVersion
 }
 
 function Get-ChocoVersion {
-    return "Chocolatey $(choco --version)"
+    return $(choco --version)
 }
 
 function Get-VcpkgVersion {
     $commitId = git -C "C:\vcpkg" rev-parse --short HEAD
-    return "Vcpkg (build from commit $commitId)"
+    return "(build from commit <$commitId>)"
```

Comment:
Please return message format back. We don't need `<>` symbols

Reply:
corrected